components/02-molecules/accordion/accordion.twig

```diff
@@ -0,0 +1,82 @@
+{#
+ # Available Variables:
+ # - accordion__heading
+ #
+ # Available Blocks:
+ # - accordion__items
+ #}
+
+{% set accordion__base_class = 'accordion' %}
+
+{% set expand_all__content %}
+  Expand All
+  {% include "@atoms/images/icons/_icon.twig" with {
+    icon__name: 'chevron-down',
+    icon__blockname: accordion__base_class,
+  } %}
+{% endset %}
+
+{% set collapse_all__content %}
+  Collapse All
+  {% include "@atoms/images/icons/_icon.twig" with {
+    icon__name: 'chevron-down',
+    icon__blockname: accordion__base_class,
+  } %}
+{% endset %}
+
+{% set accordion__attibutes = {
+  'class': bem(accordion__base_class),
+} %}
+
+<div {{ add_attributes(accordion__attibutes) }}>
+  {% if accordion__heading %}
+    {% include "@atoms/text/headings/_heading.twig" with {
+      heading__level: '2',
+      heading__blockname: accordion__base_class,
+      heading: accordion__heading,
+    } %}H
+  {% endif %}
+  {% embed "@atoms/lists/list.twig" with {
+    list__base_class: 'controls',
+    list__blockname: accordion__base_class,
+    list__attributes: {'aria-label': 'Section controls'},
```

Comment:
So I ended up implementing `aria-controls` for the button inside the `<h2>` for each accordion item per recommendations here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls 
Because aria-controls needs to target elements based on the ID it really isn't possible to add this for the expand/collapse all buttons. I think the aria-label here is appropriate with the new description I added.